### DISCUSSION TITLE
help w proxmox & netboot.xyz boot fail

### DISCUSSION THREAD

**Discussion by rtechromancer** [antonym/netboot.xyz#1367] [General]
Hi 

I followed the tutororial here https://www.youtube.com/watch?v=4btW5x_clpg to install netboot.xyz on proxmox and pxe boot install a machine.  However I get tthe error No such file or directory "(http://ipxe.org/2d0c613b)”  when I try to boot the machine.  My googlefu is not working.  Please help!  
<img width="862" alt="Screenshot 2023-12-11 at 7 21 37 PM" src="https://github.com/netbootxyz/netboot.xyz/assets/111249190/39568428-63b0-4c7a-ad90-9cbdab472b38">

**Comment by anthonyuk** [comment]
See my comment here which may also work for you (if you are self hosting).

https://github.com/orgs/netbootxyz/discussions/1371#discussioncomment-7877117
